app-emulation/sen/sen-0.5.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
```

Comment:
Could you test 3.6 as well? If you plan to maintain multiple Python packages, that would be really helpful.